lib/k8s-test-framework/src/namespace.rs

```diff
@@ -18,8 +42,8 @@ impl up_down::CommandBuilder for CommandBuilder {
                 up_down::CommandToBuild::Up => "create",
                 up_down::CommandToBuild::Down => "delete",
             })
-            .arg("namespace")
-            .arg(&self.namespace);
+            .arg("-f")
```

Comment:
Nit: I like the long form flags when encoding into scripts / programs as it is easier to to grok them without reaching for the man pages.

I realize there is probably already usage of short flags in here, but it could be nice to slowly migrate to long.